### PR TITLE
RS/YJ/RULE19-26

### PR DESCRIPTION
**Pull request by yunjoonjung-PNNL**
This rule hasn't been tested. I'll test it out when the TCD is available.

### CODE REVIEW

File: rct229/rulesets/ashrae9012019/section19/section19rule26.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        )
+
+        return {"applicable_hvac_systems_list_p": applicable_hvac_systems_list_p}
+
+    def is_applicable(self, context, data=None):
+        hvac_p = context.proposed
+        hvac_id_p = hvac_p["id"]
+        applicable_hvac_systems_list_p = data["applicable_hvac_systems_list_p"]
+
+        return hvac_id_p in applicable_hvac_systems_list_p
```
Comment (weilixu) [inline rct229/rulesets/ashrae9012019/section19/section19rule26.py:44]: This applicable check shall be at HVAC level as RDS says:
```md
For each hvac_p in applicable_hvac_systems_list_p (all other hvac systems in the P_RMI outcome is NOT_APPLICABLE): 
```

File: rct229/rulesets/ashrae9012019/section19/section19rule26.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    def create_data(self, context, data):
+        rmi_p = context.proposed
+        applicable_hvac_systems_list_p = (
+            get_hvac_systems_serving_zone_health_safety_vent_reqs(rmi_p)
+        )
+
+        return {"applicable_hvac_systems_list_p": applicable_hvac_systems_list_p}
+
+    def is_applicable(self, context, data=None):
+        hvac_p = context.proposed
```
Comment (weilixu) [inline rct229/rulesets/ashrae9012019/section19/section19rule26.py:40]: the context.proposed is rmi here, not hvac.

**Review by weilixu — APPROVED** [review]
Approved with one comment must address

File: rct229/rulesets/ashrae9012019/section19/section19rule26.py (hunk 1 above)
Comment (yunjoonjung-PNNL) [reply rct229/rulesets/ashrae9012019/section19/section19rule26.py:44]: great point! Addressed.

File: rct229/rulesets/ashrae9012019/section19/section19rule26.py (hunk 2 above)
Comment (yunjoonjung-PNNL) [reply rct229/rulesets/ashrae9012019/section19/section19rule26.py:40]: Addressed.

**Comment by JacksonJ-KC** [comment]
> This rule hasn't been tested. I'll test it out when the TCD is available.

https://github.com/pnnl/ruleset-checking-tool/pull/1085

**Comment by yunjoonjung-PNNL** [comment]
This rule was tested with TCD and passed.

**Pull request merged**